org-cyf-guides/content/contributing/minutes/index.md

```diff
@@ -17,6 +17,80 @@ Attendees:
 
 ---
 
+## 2024-10-08
+
+Attendees: Ali, Sally, Daniel
+
+### ❗Actions from last time
+
+* [x] Ali (Carry-over): Write up requirements "spectrum" into a doc for discussion
+* [ ] Ali (Carry-over): Write up projects in the Piscine repo
+* [ ] Everyone (Carry-over): Think about SDC topics and resources.
+* [x] Everyone: Think about how we may want to assess people at the end of ITP to gate entry to the Piscine.
+* [ ] Daniel: Symlink all the tooling folders.
+* [ ] Everyone: Think about starter tasks for new potential team members.
+
+### 📝 Agenda points
+
+#### Piscine availability
+
+- Ali will be away for the first week of Feb
+	- So first 2 weekends will be away :/
+	- Possibly could shift Piscine to last 3 weeks of Feb, so that Ali only misses 1 week rather than 2
```

Comment:
```suggestion
	- It just needs to start and end in Feb. Possibly could shift Piscine to last 3 weeks of Feb, so that Ali only misses 1 week rather than 2
```